docs/node-mixin/config.libsonnet

```diff
@@ -35,6 +35,16 @@
     // just a warning for K8s nodes.
     nodeCriticalSeverity: 'critical',
 
+    // Available disk space (%) thresholds on which to trigger the
+    // 'NodeFilesystemSpaceFillingUp' alerts. In some cases you'll want
+    // to adjust these, e.g. by default Kubernetes runs the image
+    // garbage collection when the disk usage reaches 85% of its
+    // available space. In that case, you'll want to reduce the 
+    // critical threshold below to something like 14 or 15, otherwise
+    // the alert could fire under normal node usage.
```

Comment:
Rather than explaining a particular use case in much detail, how about explaining better what this threshold is doing? The alerts fire if the disk usage grows in a way that it is predicted to run out in 4h or 1d _and_ if the provided thresholds have been reached right now. From the comment here alone, I would understand that the alert is just about a fixed threshold.

Reply:
I agree, I'll adjust the description